[Bug] IPosition not exported from @kitware/vtk.js/Rendering/Core/RenderWindowInteractor

### Bug description

Hello,

I am encountering a TypeScript error when trying to build my Vue project. The error message indicates that the exported variable useViewerStore has or is using the name IPosition from the external module @kitware/vtk.js/Rendering/Core/RenderWindowInteractor, but it cannot be named.

```shell
src/stores/miniStore.ts:6:14 - error TS4023: Exported variable 'useViewerStore' has or is using name 'IPosition' from external module "/Users/jeremy/poc/debug/node_modules/@kitware/vtk.js/Rendering/Core/RenderWindowInteractor" but cannot be named.

6 export const useViewerStore = defineStore('viewer', () => {
```

### Steps to reproduce

Clone https://github.com/jeremygau/debug
npm i
npm run build

### Detailed Behavior

_No response_

### Expected Behavior

The project should build without error

### Environment

- vtk.js version: @kitware/vtk.js@32.5.1
- OS: MacOS 14

--- a/Sources/Rendering/Core/RenderWindowInteractor/index.d.ts
+++ b/Sources/Rendering/Core/RenderWindowInteractor/index.d.ts
@@ -63,7 +63,7 @@ export interface IRenderWindowInteractorInitialValues {
   mouseScrollDebounceByPass?: boolean;
 }
 
-interface IPosition {
+export interface IPosition {
   type: string;
 }
 

